instaloader/instaloader.py

```diff
@@ -232,11 +232,12 @@ def __init__(self,
                  fatal_status_codes: Optional[List[int]] = None,
                  iphone_support: bool = True,
                  title_pattern: Optional[str] = None,
-                 sanitize_paths: bool = False):
+                 sanitize_paths: bool = False,
+                 proxies: dict = None):
```

Comment:
If the value can be `None` the type should reflect this through `Optional[Dict[str,str]]`.